index.js

```diff
@@ -0,0 +1,77 @@
+function validatePassword (pwd) {
+    //* Must be a minimum of 8 characters (8 characters or more) 
+    if (pwd.length < 8) {
+        return false
+    }
+    //* Must contain at least one lower case letter
+    const lowerCase = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
```

Comment:
I don't blame you for approaching your checking of lower and uppercase characters in this way (pretty sure I did it this way initially too), however as you can tell it feels unnecessarily long-winded and like there must be a shorter way of going about it, and there is, however it isn't very intuitive and would require some googling, which is what this portion of the assignment was trying to nudge you to do. One such way of going about this is the following:
`
function isLowerCase(pwd, index) {
  return pwd.charCodeAt(index) >= 97 && pwd.charCodeAt(index) <= 122
}
`